src/overlays/actors/ovl_Obj_Chan/z_obj_chan.h

```diff
@@ -5,12 +5,56 @@
 
 struct ObjChan;
 
+#define OBJCHAN_SUBTYPE(this) (((this)->actor.params >> 12) & 0xF)
```

Comment:
```suggestion
#define OBJCHAN_SUBTYPE(thisx) (((thisx)->params >> 12) & 0xF)
```
Params macros should take the base actor pointer as there argument. Would require changing any uses of 
`OBJCHAN_SUBTYPE(this)` -> `OBJCHAN_SUBTYPE(&this->actor)`.